src/components/select/select.spec.js

```diff
@@ -333,6 +333,78 @@ describe('<md-select>', function() {
 
     }));
 
+    describe('md-select-header behavior', function() {
+      it('supports rendering a md-select-header', inject(function($rootScope, $compile) {
+        $rootScope.val = [1];
+        var select = $compile('<md-input-container>' +
```

Comment:
Random note for future reference (you totally don't need to change it here): most code formatters will ignore  the lovely spacing you applied below and align all of the quotation marks up with one another. I've started putting my spacing _inside_ the quotes to avoid this.

**Examples for clarification:**

Instead of

```js
'<div>' +
  'some text' +
'</div>'
```

I generally do this


```js
'<div>' +
'  some text' +
'</div>'
```

It helps me not get frustrated when I do a `Ctrl-Shift-L` later and reformat all that pretty code :smile: